### PR TITLE
1817DE Alpha

### PR DESCRIPTION
**Pull request by scottredracecar**

### CODE REVIEW

File: lib/engine/game/g_1817_de/meta.rb [hunk 1]
```diff
@@ -8,15 +8,17 @@ module G1817DE
...
-        DEV_STAGE = :prealpha
+        DEV_STAGE = :alpha
         DEPENDS_ON = '1817'
 
         GAME_DESIGNER = 'Scott Petersen'
         GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/1817DE'
         GAME_LOCATION = 'Germany'
         GAME_RULES_URL = {
-          '1817DE' =>
+          '1817DE Rule Differences' =>
```
Comment (jenf) [inline lib/engine/game/g_1817_de/meta.rb:18]: is this supposed to be locked?

Comment (scottredracecar) [reply lib/engine/game/g_1817_de/meta.rb:18]: Okay, wiki link is live now.

File: lib/engine/game/g_1817_de/meta.rb [hunk 2]
```diff
@@ -8,15 +8,17 @@ module G1817DE
...
                           'https://docs.google.com/document/d/1lH3TcQc6etptZDWMbX1oZe7ne2VbbinAYkX8W_lpu-c/edit',
```
Comment (michaeljb) [inline lib/engine/game/g_1817_de/meta.rb:19]: @scottredracecar I believe @jenf meant this link, it still seems to lock people out - https://docs.google.com/document/d/1lH3TcQc6etptZDWMbX1oZe7ne2VbbinAYkX8W_lpu-c/edit

Looks like we could use https://github.com/tobymao/18xx/wiki/1817DE instead?

Comment (scottredracecar) [reply lib/engine/game/g_1817_de/meta.rb:19]: Yes, that makes more sense. Fixed it now.

**Pull request merged**